clients/python/test/e2e/test_notebooks.py

```diff
@@ -45,3 +46,90 @@ def test_run_notebooks(tmp_path: Path, notebook: Path, params: dict[str, Any] =
         kernel_name="python3",
         parameters=params,
     )
+
+
+def get_tutorials(osparc_version: Optional[str] = None) -> List[Path]:
```

Comment:
TIP: I think there is a `packaging.version.Version` object

Reply:
Yes, I know about that one. However, since here I use the input to extract something from a `dict` (loaded from a json). So I think it is makes sense to simply use the string here.